portfolio/src/main/webapp/mapScript.js

```diff
@@ -11,10 +11,159 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
+let map;
+let editMarker
 
 function createMap() {
-  const map = new google.maps.Map(
+  const home_coords = { lat: 26.0321, lng: -80.2627 };
+  map = new google.maps.Map(
     document.getElementById('map'),
-    { center: { lat: 26.0321, lng: -80.2627 }, zoom: 16 });
+    { center: home_coords, zoom: 16,
+    styles: [
```

Comment:
this whole styles object seems like a constant, it may be helpful to move it outside of the function as a const and reference it for the Map constructor.